File: back-end/src/main/java/com/backend/model/attempt/GetAttemptModel.java
package com.backend.model.attempt;

import lombok.*;

import java.sql.Timestamp;
import java.util.List;

@NoArgsConstructor
@AllArgsConstructor
@Setter
@Getter
@ToString
public class GetAttemptModel {

    private Integer id;
    private String content;
    private String description;
    private String type;
    private Boolean status;
    private String historyStatus;
    private Timestamp startDate;
    private Timestamp endDate;
    private List<GetAttemptAnswerModel> answers;

}

File: front-end/src/app/_models/getAttemptModel.ts
import {GetAttemptAnswerModel} from './getAttemptAnswerModel';

export class GetAttemptModel {
  id: number;
  content: string;
  description: string;
  status: boolean;
  historyStatus: string;
  type: string;
  startDate: Date;
  endDate: Date;
  answers: GetAttemptAnswerModel[];
}
